algorithms/src/std_algorithms/numeric/Kokkos_InclusiveScan.hpp

```diff
@@ -57,6 +57,30 @@ namespace Kokkos {
 namespace Experimental {
 namespace Impl {
 
+template <class ExeSpace, class IndexType, class ValueType, class FirstFrom,
+          class FirstDest>
+struct InclusiveScanDefaultFunctorNoInit {
```

Comment:
similar comment as for exclusive scan, I would rename this: `InclusiveScanDefaultFunctorForKnownIdentityElement`